fgpyo/tests/test_sequence.py

```diff
@@ -15,3 +16,11 @@ def test_reverse_complement() -> None:
 
     with pytest.raises(KeyError):
         reverse_complement("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+
+def test_gc_content() -> None:
+    assert gc_content("ATATATATTATATA") == 0.0
+    assert gc_content("GCGCGCGCGCGCG") == 1.0
+    assert gc_content("ACGT") == 0.5
+    assert gc_content("") == 0.0
+    assert gc_content("ACGTN") == 0.4
```

Comment:
1. Can you make these parameterized with pytest?
2. After (1), then can you test both upper and lower case (you can just have two asserts in the test method, one that upper-cases explicitly, and the other that lower-cases explicitly)

Reply:
1. parameterized with pytest
2. unit test includes mixed base cases